packages/application-shell/src/apollo-links/header-link.ts

```diff
@@ -19,8 +17,10 @@ import * as oidcStorage from '../utils/oidc-storage';
 
 declare let window: ApplicationWindow;
 
-type ApolloContextWithInMemoryCache = TApolloContext & {
-  cache: ApolloClient<NormalizedCacheObject>;
+type ApolloContextWithInMemoryCache = TApolloContext;
```

Comment:
Just wanted to double check here because the new type is named as it has some kind of cache property but we're not extending the `TApolloContext` withe the `cache` property anymore and [it doesn't seem to have it](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/application-shell/src/utils/apollo-context.ts#L11) by itself.

Reply:
Good catch, I'll remove this type yes.